bin/deploy-translation.js

```diff
@@ -0,0 +1,13 @@
+#! /usr/bin/env node
+var doctoc = require("doctoc");
+var shell = require("shelljs");
+var glob = require("glob");
+
+/*
+    Generate menu for all ReadMe with DocToc
+ */
+glob("./README*.md", function (er, files) {
+    files.forEach(function (item) {
+        shell.exec("doctoc " + item + " --github");
```

Comment:
What about to change code like `doctoc *.md --github`

Reply:
I thought about it. But the repository may also other .md files (like CONTRIBUTE.md) that do not need to be processed with DocToc.